apps/shell/tash_command.c

```diff
@@ -283,6 +283,8 @@ int tash_cmd_install(const char *str, TASH_CMD_CALLBACK cb, int thread_exec)
 	int cmd_idx;
 
 	if (TASH_MAX_COMMANDS == tash_cmds_info.count) {
+		printf("Allowed Max tash cmds: %d and Current tash cmd count: %d\n", TASH_MAX_COMMANDS, tash_cmds_info.count);
```

Comment:
tash_cmds_info.count is not increasing due to return -1. How do you print as you show it?

Reply:
Sorry. local changes were not part of push..